client-rpc/src/handler.rs

```diff
@@ -49,6 +49,16 @@ impl RpcHandler {
     ) -> Result<Self> {
         let mut io = IoHandler::new();
         let storage = SledStorage::new(&storage_dir)?;
+
+        let polling_storage = storage.clone();
+        std::thread::spawn(move || {
+            loop {
+                polling_storage.flush().expect("sled storage flush");
+                // every 1 second
+                std::thread::sleep(std::time::Duration::from_secs(1));
+            }
+        });
```

Comment:
sled has this config: https://github.com/spacejam/sled/blob/master/src/config.rs#L394

that doesn't work?

Reply:
Oh I see the commit comment
> (sled Config.flush_every_ms didn't work)

Reply:
strange, that should work, i'll try again